### PR TITLE
Add skeleton of Huawei Cloud plugin

### PR DESCRIPTION
**Pull request by niuzhenguo**
Related #21

### CODE REVIEW

**Review by iskey — COMMENTED** [review]
Just a note (maybe it is not a good way to move OrchestratorPlugin.java this to ../plugins).

**Comment by jbonofre** [comment]
@iskey not sure I got your point.

IMHO, `OrchestratorPlugin` class should stay in `orchestrator` service.

For me, each plugin in a new service; so, I would expect to have (in terms of Maven modules):

* `services/huaweicloud`
* `services/openstack`
* ...

File: services/orchestrator/src/main/java/org/eclipse/osc/orchestrator/plugins/huaweicloud/HuaweiCloudPlugin.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package org.eclipse.osc.orchestrator;
+
+import lombok.extern.java.Log;
+import org.apache.karaf.minho.boot.spi.Service;
+import org.eclipse.osc.services.ocl.loader.Ocl;
+
+@Log
+public class HuaweiCloudPlugin implements OrchestratorPlugin, Service {
```
Comment (jbonofre) [inline services/orchestrator/src/main/java/org/eclipse/osc/orchestrator/plugins/huaweicloud/HuaweiCloudPlugin.java:8]: `HuaweiCloudPlugin` should be in its own module, not directly in the `Orchestrator`.

I think it's better to create `services/huaweicloud`. This module will have `orchestrator` as dependency and implements `HuaweiCloudPlugin`.

I'm doing the same with `services/openstack`.

Like this, we have a complete extensible mechanism where users can provide their own plugins.

It's in the runtime that we define the plugins we want (in the assembly).
We can imagine to have several runtime (like `osc-huaweicloud`, `osc-opentstack`, `osc-aws`, ...).

**Review by jbonofre — CHANGES_REQUESTED** [review]
`OrchestratorPlugin` should stay in `Orchestrator` module and we should have a dedicated service/module for HuaweiCloud (e.g. `services/huaweicloud`).

**Comment by jbonofre** [comment]
@niuzhenguo @iskey you can see the `openstack` plugin service skeleton on PR #29 

It's what I meant by having a `huaweicloud` plugin service.

**Comment by iskey** [comment]
> @iskey not sure I got your point.
> 
> IMHO, `OrchestratorPlugin` class should stay in `orchestrator` service.
> 
> For me, each plugin in a new service; so, I would expect to have (in terms of Maven modules):
> 
> * `services/huaweicloud`
> * `services/openstack`
> * ...

Yes， I think maven modules are better.

**Comment by niuzhenguo** [comment]
@jbonofre @iskey I will follow the openstack plugin way, we can do more discussion there :P

**Comment by iskey** [comment]
lgtm.

**Review by iskey — APPROVED** [review]

**Review by jbonofre — APPROVED** [review]

**Pull request merged**